packages/block-library/src/image/index.js

```diff
@@ -216,15 +217,59 @@ export const settings = {
 			/>
 		);
 
-		return (
-			<figure className={ classes }>
+		const figure = (
+			<Fragment>
 				{ href ? <a href={ href }>{ image }</a> : image }
 				{ caption && caption.length > 0 && <RichText.Content tagName="figcaption" value={ caption } /> }
+			</Fragment>
+		);
+
+		if ( 'left' === align || 'right' === align || 'center' === align ) {
+			return (
+				<div>
```

Comment:
Wasn't this meant to be an aside (from the PR description)?

Reply:
Yes, but per discussion in https://github.com/WordPress/gutenberg/pull/7721#issuecomment-408772694, I reverted to using a div. I will update the description.

Reply:
I'm also curious about this though, because it doesn't have a `className` attribute yet in the fixtures the tests expect/output a `<div>` with a `class` attribute. See: 66f6f01a70bd725b3bcada40776156fa86b0cb9d

I'm a bit confused by that...